### PR TITLE
Task 5 Integration with S3

### PR DESCRIPTION
**Pull request by asis-shukla**
### What is Done

- [x] Task 5.1
- [x] Task 5.2
- [x] Task 5.3
- [x] Task 5.4

### Additional (optional) tasks:

- async/await is used in lambda functions
- At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into a new folder in the same bucket called parsed, and then deleted from uploaded folder) 

### URL:
GET: _https://omes0a6rh3.execute-api.us-east-1.amazonaws.com/import/{fileName}_
CloudFront URL: _https://dcclx935ir039.cloudfront.net/_

### PR Link:
BE: _https://github.com/asis-shukla/shop-be-aws/pull/3_
FE: _https://github.com/asis-shukla/shop-react-redux-cloudfront/pull/3_

**Self Evaluation: 5 + 2 = 7**

### CODE REVIEW

File: import-service/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  environment:
+    FILE_UPLOAD_BUCKET_NAME: ${self:custom.fileBucketName}
+  iam:
+    role:
+      statements:
+        - Effect: "Allow"
+          Action: "s3:*"
+          Resource: "arn:aws:s3:::${self:custom.fileBucketName}"
+
+        - Effect: "Allow"
```
Comment (Sergey-Churilin) [inline import-service/serverless.yml:23]: Looks like a duplication of previous statement on line 19

**Comment by Sergey-Churilin** [comment]
Looks good to me, 7\8

File: import-service/serverless.yml (hunk 1 above)
Comment (asis-shukla) [reply import-service/serverless.yml:23]: above is for the bucket and the below one is for files inside the bucket. Do we need to do that or allowing bucket automatically allow to all objects inside bucket?

Comment (Sergey-Churilin) [reply import-service/serverless.yml:23]: you are right, I missed difference in last line. We need both.
For the line 20 as value I used s3:ListBucket, not just *.

**Pull request merged**